### ISSUE TITLE
Hard coded Path to Users/Tony

### ISSUE THREAD

**Issue by LanceMcCarthy** [event-streams-dotnet/event-stream-processing#1]
Hi team, while building the Sample.Worker project, it appears there is a hard coded file path to Tony's source control folder:

![image](https://user-images.githubusercontent.com/3520532/95886777-07d56300-0d4d-11eb-8450-249888500221.png)

## Cause

After further investigation, it seems these are only hard coded in the csproj and not in code.

![image](https://user-images.githubusercontent.com/3520532/95886898-2d626c80-0d4d-11eb-9a43-dcc5f1bf9c32.png)

## Recommended solution

Use relative path, or just the file name directly (no need for base path at all)

```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>netstandard2.0</TargetFramework>
    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
    <Authors>Tony Sneed</Authors>
    <Company>Tony Sneed</Company>
    <Description>Event stream processing micro-framework.</Description>
    <PackageLicenseExpression>MIT</PackageLicenseExpression>
    <PackageProjectUrl>https://github.com/event-streams-dotnet/event-stream-processing</PackageProjectUrl>
    <PackageIcon>event-streams-icon.png</PackageIcon>
    <PackageTags>event-streams event-stream-processing apache-kafka kafka</PackageTags>
  </PropertyGroup>

  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
    <DocumentationFile>EventStreamProcessing.Abstractions.xml</DocumentationFile>
  </PropertyGroup>

  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
    <DocumentationFile>EventStreamProcessing.Abstractions.xml</DocumentationFile>
  </PropertyGroup>

  <ItemGroup>
    <None Include="..\..\images\event-streams-icon.png">
      <Pack>True</Pack>
      <PackagePath></PackagePath>
    </None>
  </ItemGroup>

</Project>
```

**Comment by LanceMcCarthy** [comment]
There were some other stragglers in the Kafka project as well. I just opened https://github.com/event-streams-dotnet/event-stream-processing/pull/2 to take care of both at the same time

**Comment by tonysneed** [comment]
Fixed by #2.
